src/Sylius/Bundle/ApiBundle/Validator/Constraints/CanPaymentMethodBeChanged.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/** @experimental */
+final class CanPaymentMethodBeChanged extends Constraint
+{
+    public string $cannotChangePaymentMethodForCancelledOrder = 'sylius.payment_method.cannot_change_payment_method_for_cancelled_order';
```

Comment:
```suggestion
    public string $cannotChangePaymentMethodForCancelledOrder = 'sylius.payment_method.cannot_change_payment_method_for_cancelled_order';
```

What is it?

Reply:
It is a validation message used in validator

Reply:
So, it should be a constant, not a public field.